docs/uma-tokenholders/ulabs-dappmining.md

```diff
@@ -0,0 +1,20 @@
+---
+title: uLABS dApp Mining Program
+sidebar_label: dApp Mining
+---
+
+uLABS, the product arm of UMA, shepherds the yield dollar token contracts and manages the liquidity mining program. Half of the rewards for the Yield Dollar tokens are allocated for liquidity mining, with the other half unspent.
```

Comment:
Agreed. I also think listing dapp mining interfaces is redundant when you have them under liquidity mining as well. 

I like the direction that you were going here @henrystats, but I agree with Tom. I think these sections should be more of explainers rather than stat pages. We should be able to reuse a lot of the great content that you have, but we need to do some more thinking on the organization here.